map/src/menu/actions/SortActions.jsx

```diff
@@ -78,7 +79,28 @@ function byLocation(files, reverse, markers = null) {
     });
 }
 
-const allMethods = {
+export function sort({ method, setSortFiles, setSortGroups, markers, files, groups, favoriteGroup }) {
```

Comment:
Please rename function name `sort()` to something more detailed (to avoid confusing with standard JS .sort() calls).

doSort, sortFilesGroups, etc